about.html

```diff
@@ -0,0 +1,148 @@
+        </div>
+        <div class="invitation">
+            <p>Please contact us through email for futher </p>
+            <p>questions about SANO</p>
+            <p class="underline"><strong>support@sano.org</strong></p>
+        </div>
+    </div>
+     <!-- about section  -->
+    <div class="logo" id>    
+        <div class="logo_content">
+            <h4>ABOUT SANO LOGO</h4>
+            <img src="./images/line2.png" alt="line" width="160px" height="20px">
+            <p class="logop"><strong>Lorem, ipsum dolor sit amet consectetur adipisicing elit. Soluta molestias nam similique numquam amet magni,<br> 
+                voluptates facere consequuntur<br> 
+                fugit aliquid illo incidunt quod accusantium
+                 laudantium nesciunt. Itaque, quis. Temporibus, 
+                 ad!
+            </strong></p>
+                 <img class="aboutlogo" src="./images/thelogo (1).png" alt="logo" width="250px" height="250px">
+        </div>    
+    </div>
+
+    <!-- past section  -->
+    <div class="past_content">
+
+        <ul class="gallery">
+            <li class="group">
+                <img src="./images/group1.jpg" alt="group1">
+              <div class="group-info">
+                <h2 class="past-year">2017</h2>
+                <p>SANO sumit in Lagos 2017</p>
+              </div>
+            </li>
+            <li class="group">
+                <img src="./images/group2.jpg" alt="group2">
+                <div class="group-info">
+                    <h2 class="past-year">2018</h2>
+                    <p>SANO sumit in Abuja 2018</p>
+                  </div>
+            </li>
+          </ul>
+
+    </div>
+ <!-- partner section  -->
+    <div class="partner">
+        <div class="partners">
+            <h2>Partners</h2>
+            <img src="./images/line2.png" alt="line" width="50px">
+        </div>
+            <div class="row1">
+                <h2><strong>mozilla</strong></h2>
+                <h3>Google</h3>
+                <h1><strong>NAVER</strong></h1>          
+            </div>
+            <div class="row2">
+                <h3>daumkakao</h3>
+                <h3><span><img src="./images/airbnb.png" alt="airbnb" width="36px" height="36px"></span>airbnb</h3>
+            </div>
+    </div>
+
+    <!-- footer section  -->
+    <footer>
+        <div class="footer">
+            <img src="./images/logo1.png" alt="logo1" width="180px">
+            <div class="logo_contents">
+                <h4>SANO UNICORN</h4>
+                <p><strong>2022:26:11(SAT) ~ 27(SUN)</strong></p>
+                <p>Lorem, ipsum dolor sit amet consectetur adipisicing elit. Sunt illum fugit, molestiae vero eligendi commodi nihil officia. Ab totam eaque, cupiditate perspiciatis asperiores tenetur, est eos officiis quae, commodi modi.</p>
+            </div> 
+        </div>   
+    </footer>
```

Comment:
- [ ] The background color of the footer on the about page should be darker, kindly consider making the changes so the footer looks like the provided design

|Your page|CC design|
|-|-|
|![Screenshot_20221123_200427](https://user-images.githubusercontent.com/25227022/203627741-66712d03-a14a-456a-89dc-88c0813905a8.png)|![Screenshot_20221123_200810](https://user-images.githubusercontent.com/25227022/203628010-94203599-8b5c-4e6c-bff7-c507609b6b68.png)|